site/en/blog/origin-trials/index.md

```diff
@@ -75,6 +75,15 @@ the [deprecation trial to continue using AppCache](/origintrials/#/view_trial/17
 1. When your token expires, you will get an email with a renewal link.
    To do so, you are again asked to submit feedback.
 
+{: #multiple}
+
+You can register for the same origin trial multiple times, for different origins, and include 
+multiple tokens in the same page if necessary. This can be useful if you need to provide tokens 
+that are valid for resources served from different origins, such as code included on multiple 
+sites that you own. The origin trials framework looks for the first valid token and then
+ignores all other tokens. You can validate this
+[from Chrome DevTools](/blog/origin-trial-troubleshooting/#use-chrome-devtools-to-check-tokens).
```

Comment:
```suggestion
You can register for the same origin trial multiple times, for different origins, and include 
multiple tokens in the same page. This can be useful if you need to provide tokens 
that are valid for resources served from different origins, such as code included on multiple 
sites that you own.

The origin trials framework looks for the first valid token and then
ignores all other tokens. You can validate this
[with Chrome DevTools](/blog/origin-trial-troubleshooting/#use-chrome-devtools-to-check-tokens).
```